models/blank_space/src/training/train_model.py

```diff
@@ -16,45 +16,37 @@
 from get_data import get_partition_data
 
 
-def train(model_config, para_config):
-    print("Training...")
-    if not model_config["sweep"]:
-        PATH_RAW, _, _ = setup_data_paths(PATH)
-        PATH_ARTIFACTS = setup_artifacts_paths(PATH)
-        dataset = pd.read_parquet(PATH_RAW / 'raw.parquet')
-        if model_config["algorithm"] == "HurdleRegression":
-            model = HurdleRegression(clf_name=model_config["clf_name"], reg_name=model_config["reg_name"], clf_params=para_config["clf"], reg_params=para_config["reg"])
-        else:
-            model = globals()[model_config["algorithm"]](**para_config)
-        # print(model)
-            
-        # Train partition
+def get_model(model_config, para_config):
```

Comment:
Do we have a config called para_config? It's the config_hyperparameters right? If so, just use the full name here